### DISCUSSION TITLE
Convergence with Focei algorithm

### DISCUSSION THREAD

**Discussion by pg320** [nlmixr2/rxode2#519] [General]
Probably a really dumb question. How to increase convergence when solving with Focei (saem doesn't work for this problem). 

I'm looking for something like the equivalent of nBurn and nEm used in Saem control.

**Comment by mattfidler** [comment]
If you want to increase the convergence criterion during solving you can increase the accuracy of the solve by `foceiControl(sigdigs=12)` which controls the ODE solving tolerances, as well as the general optimization tolerances.  

If you want to try another algorithm for minimization you could always change it, for example `foceiControl(outerOpt="bobyqa")` gives a good derivative free optimization method for the outer problem in focei.
